packages/core/truffle-test/scripts/Voting.js

```diff
@@ -1,394 +0,0 @@
-const VotingScript = require("../../scripts/Voting.js");
```

Comment:
why was this removed?

Reply:
It was a broken script  (Voting.js doesn't even exist in the repo anymore)